### DISCUSSION TITLE
convert from workflow to yaml

### DISCUSSION THREAD

**Discussion by uditrugman** [codemagic-ci-cd/codemagic-docs#1523] [Q&A]
Hi,
is it possible to start with the workflow editor and then export as codemagic.yaml?

thanks

**Comment by himesh-cm** [comment]
Hi @uditrugman,
Currently, it is not possible to export the workflow configuration as `yaml` file.

You can refer to our sample projects for flutter workflow [here](https://github.com/codemagic-ci-cd/codemagic-sample-projects/tree/main/flutter/flutter-android-and-ios-yaml-demo-project). It should work with minimal changes.
Feel free to contact us if you need help with the `yaml`.

**Reply by edgarfroes** [reply]
Sorry to resurrect this discussion, but is this still not a feature? It'd be very good to have it since my workflows are made with the editor but have some complexity in them.

**Comment by malbolged** [comment]
@himesh-cm Hi!
will this feature be unvailable forever ?
or is there a plan to implement it ?

**Reply by malbolged** [reply]
@himesh-cm An answer would be much appreciated, please.
Thank you!

**Comment by TheCarpetMerchant** [comment]
The fact this is still not possible is genuinely baffling. What is the technical reason, if any, preventing this from being 4 enginner-hours of work ?

**Comment by John-Daniels** [comment]
@uditrugman Created a simple solution for this here https://github.com/John-Daniels/codemagic-to-yaml 
The reason why its not exportable is beacause resources in workflow cant be transferred directly to the yaml like secrets and keystores and appstore configs, I've found a way to do a full coversion of your workflow you plus how to setup the other stuff like appstore, playstore and other configurations. [here](https://john-daniels.github.io/codemagic-to-yaml/)
